tox.ini

```diff
@@ -64,72 +32,65 @@ deps =
     pexpect==4.8.0
     pytest-rerunfailures==9.0
 
-commands =
-    pip install .[all]
-    pip install -i https://test.pypi.org/simple/ fetch-p2p-api==0.0.2
-    pytest -rfE --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=xml --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
+commands = pip install -i https://test.pypi.org/simple/ fetch-p2p-api==0.0.2
+           pytest -rfE --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=xml --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
 
 [testenv:py3.6]
 basepython = python3.6
-passenv = *
-deps =
-    pytest==5.3.5
-    pytest-cov==2.8.1
-    pytest-asyncio==0.10.0
-    pytest-randomly==3.2.1
-    docker
-    colorlog==4.1.0
-    defusedxml==0.6.0
-    oef==0.8.1
-    gym==0.15.6
-    numpy==1.18.1
-    tensorflow==1.14.0
-    vyper==0.1.0b12
-    openapi-core==0.13.2
-    openapi-spec-validator==0.2.8
-    black==19.10b0
-    mistune==2.0.0a4
-    aiohttp==3.6.2
-    SQLAlchemy==1.3.16
-    pynacl==1.3.0
-    pexpect==4.8.0
-    pytest-rerunfailures==9.0
 
+[testenv:py3.7]
+basepython = python3.7
 
-commands =
-    pip install .[all]
-    pip install -i https://test.pypi.org/simple/ fetch-p2p-api==0.0.2
-    pytest -rfE --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=xml --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
+[testenv:py3.8]
+basepython = python3.8
 
 [testenv:bandit]
+skipsdist = True
+skip_install = True
 deps = bandit==1.6.2
 commands = bandit -s B101 -r aea benchmark examples packages scripts tests
 
 [testenv:black]
+skipsdist = True
+skip_install = True
 deps = black==19.10b0
 commands = black aea benchmark examples packages scripts tests
 
 [testenv:black-check]
+skipsdist = True
+skip_install = True
 deps = black==19.10b0
 commands = black aea benchmark examples packages scripts tests --check --verbose
 
 [testenv:copyright_check]
+skipsdist = True
+usedevelop = False
+skip_install = True
+deps =
 commands = {toxinidir}/scripts/check_copyright_notice.py --directory {toxinidir}
 
 [testenv:hash_check]
-deps = python-dotenv
+skipsdist = True
+usedevelop = True
+deps = ipfshttpclient
 commands = {toxinidir}/scripts/generate_ipfs_hashes.py --check {posargs}
 
 [testenv:package_version_checks]
-deps = python-dotenv
+skipsdist = True
+usedevelop = True
+deps =
```

Comment:
the only dependency is the `aea` package, installed with source installation via `usedevelop = True` and `skip_install = False` (by default)